deepchem/models/tensorgraph/models/unet.py

```diff
@@ -0,0 +1,74 @@
+"""
+U-Net implementation
+ U-Net: Convolutional Networks for Biomedical Image Segmentation,
+ Olaf Ronneberger, Philipp Fischer, Thomas Brox
+
+ https://arxiv.org/pdf/1505.04597.pdf
+
+"""
+import tensorflow as tf
+import deepchem as dc
+from deepchem.models import Sequential
+from deepchem.models.tensorgraph.layers import Conv2D, MaxPool2D, Conv2DTranspose, Concat, Feature
+from deepchem.models import TensorGraph
+
+class UNet(TensorGraph):
+    """
+        U-Net architecture implementation.
+        Parameters
+        ----------
+        img_rows : int
+         number of rows of the image.
+        img_cols: int
+         number of columns of the image
+    """
+    def __init__(self,img_rows=512, img_cols=512, model=dc.models.TensorGraph(), **kwargs):
+        super(UNet, self).__init__(use_queue=False, **kwargs)
+        self.img_cols = img_cols
+        self.img_rows = img_rows
+        self.model = dc.models.TensorGraph()
+
+        input = Feature(shape=(None, self.img_rows, self.img_cols))
```

Comment:
input is a python keyword.  Can we also put the graph building in a method
```python
def build_graph(self):
```
like the Graph Models?

This will help for consistency.